cloud_governance/aws/zombie_non_cluster/run_zombie_non_cluster_policies.py

```diff
@@ -167,12 +170,12 @@ def _trigger_mail(self, tags: list, resource_id: str, days: int, resource_type:
                                                                     notification_days=self.DAYS_TO_TRIGGER_RESOURCE_MAIL,
                                                                     delete_days=self.DAYS_TO_DELETE_RESOURCE,
                                                                     resource_name=resource_name, resource_id=resource_id,
-                                                                    resource_type=resource_type, msgadmins=self.DAYS_TO_NOTIFY_ADMINS)
+                                                                    resource_type=resource_type, msgadmins=self.DAYS_TO_NOTIFY_ADMINS, extra_purse=kwargs.get('extra_purse'))
             if not kwargs.get('admins'):
-                self._mail.send_email_postfix(to=to, content=body, subject=subject, cc=cc, resource_id=resource_id, message_type=kwargs.get('message_type'))
+                self._mail.send_email_postfix(to=to, content=body, subject=subject, cc=cc, resource_id=resource_id, message_type=kwargs.get('message_type'), extra_purse=kwargs.get('extra_purse'))
```

Comment:
I think that extra_purse should be estimate_cost in all the place

Reply:
Basically, we pay extra for not using that resources. So I put extra_purse.

Reply:
Ok